sample/ios/Podfile

```diff
@@ -10,7 +10,7 @@ target 'sample' do
   use_react_native!(
     :path => config["reactNativePath"],
     # to enable hermes on iOS, change `false` to `true` and then install pods
-  	:hermes_enabled => true
+  	:hermes_enabled => false
```

Comment:
Do we need this change?

Reply:
Nope, this got accidentally committed, i was using a debugger and it doesn't work with iOS hermes. Will revert.